### DISCUSSION TITLE
Aircrack-ng on cluster?

### DISCUSSION THREAD

**Discussion by BITIW** [aircrack-ng/aircrack-ng#2593] [Q&A]
Greetings everyone! I want to ask this question - is Aircrack-ng adapted for use on a cluster (like k8s)? If not, then maybe there is some software that will crack handshake on the cluster?

**Comment by Mister-X-** [comment]
Aside from requiring some development to sync the different instances (split wordlist, and shutdown when found), it should work just fine.

**Reply by BITIW** [reply]
So if I run on a cluster, they will work as one, right?

**Reply by Mister-X-** [reply]
It is not designed to work in parallel without extra work. You can make it, but it requires you to develop something to handle that.
